src/ServiceBouncer/MainForm.cs

```diff
@@ -15,22 +17,33 @@ public partial class MainForm : Form
         private readonly List<ServiceViewModel> services;
         private bool isActive;
         private string machineHostname;
-        private int backgroundRefreshSeconds = 1;
+        private int backgroundRefreshSeconds;
 
-        public MainForm()
+        public MainForm(string machine = null)
         {
             InitializeComponent();
             isActive = true;
-            machineHostname = Environment.MachineName;
+            backgroundRefreshSeconds = 1;
+            machineHostname = machine ?? Environment.MachineName;
```

Comment:
Pass the environment.machine into the constructor.
And then the ctor doesn't need an optional value